tests/counterpoll_test.py

```diff
@@ -221,6 +221,21 @@ def test_update_route_counter_interval(self):
         assert result.exit_code == 2
         assert expected in result.output
 
+    def test_delay(self):
```

Comment:
this test is not taking into account the fast-reboot behavior. That's the part which is being fixed in this PR, so can we have a way to validate this (via UT) change in fast-reboot context? Check if delay happens when config db contains the FLEX_COUNTER_DELAY_STATUS flag.

Reply:
@vaibhavhd I am not aware of UT that invokes full fast boot flow, it's not a UT by definition. That's more an integration VS test and I am not sure either whether we have fast-boot tests on VS. Only option I see is to add a check to existing sonic-mgmt advanced-reboot test.

Reply:
I did not mean to add a UT for a full fast-reboot flow. You are right that it would be covered in the integration test.

However, my ask is for a UT "when config db contains the FLEX_COUNTER_DELAY_STATUS flag." That part can be covered in UT, IMO. What do you think?